commons/src/types/Availability.ts

```diff
@@ -36,6 +33,7 @@ export interface Manifest extends NylasManifest {
   start_date: Date;
   start_hour: number;
   view_as: "schedule" | "list";
+  timezone: string;
```

Comment:
is `timezone` too vague?

I think it's okay right? DevUsers will know intuitively that if they're setting timezone that users will see the set one along with the user's one. 

What do you think? mb we can ask Hazik if this is a bad experience

Reply:
I think as long as this is in the documentation, people will understand